visualstimulation/plot.py

```diff
@@ -1,96 +1,54 @@
-import numpy as np
+from sklearn.preprocessing import minmax_scale
```

Comment:
Since the normalization function is quite simple, I suggest to use the implementation you had earlier (moved to a separate function). The reason for that is because we don't have any other dependencies on sklearn and generally it is a good idea to keep the number of dependencies low.

Reply:
Done